dom/events/scrolling/scrollend-event-handler-content-attributes.html

```diff
@@ -43,8 +43,9 @@
   element_scrollend_arrived = true;
 }
 
+let document_scrollend_arrived = false;
 function failOnScrollEnd(event) {
-  assert_true(false, "Scrollend should not be called on: " + event.target);
+  document_scrollend_arrived = true;
```

Comment:
It'd be good to test that `event.target === document.body`, since that's the actual spec change being made in https://github.com/whatwg/html/pull/8461. (Before, it was `event.target === document`.)